<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kayıt Ol</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f0f0f0;
}
QLabel {
    color: #333333;
}
QLineEdit {
    padding: 8px;
    border: 1px solid #cccccc;
    border-radius: 4px;
    background-color: white;
}
QPushButton {
    padding: 8px;
    border-radius: 4px;
    background-color: #2196F3;
    color: white;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #1976D2;
}
QPushButton#geri_button {
    background-color: #757575;
}
QPushButton#geri_button:hover {
    background-color: #616161;
}</string>
  </property>
  <widget class="QLabel" name="baslik_kayit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kayıt Ol</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ad">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ad:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kayit_ad">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Adınızı giriniz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_soyad">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Soyad:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kayit_soyad">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Soyadınızı giriniz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>E-posta:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kayit_email">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>E-posta adresinizi giriniz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_kullaniciadi">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kullanıcı Adı:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kayit_kullaniciadi">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Kullanıcı adınızı giriniz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sifre">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Şifre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kayit_sifre">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Şifrenizi giriniz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sifre_tekrar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Şifre Tekrar:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kayit_sifre_tekrar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Şifrenizi tekrar giriniz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kayit_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Kayıt Ol</string>
   </property>
  </widget>
  <widget class="QPushButton" name="geri_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>560</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Geri Dön</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui> 